plugin/builder/algolia_menu/element.php

```diff
@@ -15,6 +15,10 @@
                 $facets[] = $facet['field'];
             }
 
+            if (empty($facets)) {
+                $facets[] = " ";
```

Comment:
come mai deve esserci per forza un facet?

Reply:
Perchè si aspetta un array di stringhe, se è vuoto non trova nulla e va in errore